app/core/settings/app.py

```diff
@@ -0,0 +1,51 @@
+from typing import List, Dict, Union
+
+from pydantic import PostgresDsn, SecretStr, validator, Field
+
+from app.core.settings.base import BaseAppSettings
+
+
+class AppSettings(BaseAppSettings):
+    debug: bool = False
+    docs_url: str = "/docs"
+    openapi_prefix: str = ""
+    openapi_url: str = "/openapi.json"
+    redoc_url: str = "/redoc"
+    title: str = "FastAPI example application"
+    version: str = "0.0.0"
+
+    database_url: PostgresDsn
```

Comment:
I think this also needs to be changed in the GitHub Actions workflow files so that they complete successfully.